test/logoutresponse_test.rb

```diff
@@ -30,36 +26,125 @@ class RubySamlTest < Minitest::Test
       end
     end
 
-    describe "#validate" do
+    describe "#validate!" do
       it "validate the response" do
         in_relation_to_request_id = random_id
 
         logoutresponse = OneLogin::RubySaml::Logoutresponse.new(valid_response({:uuid => in_relation_to_request_id}), settings)
 
-        assert logoutresponse.validate
+        assert logoutresponse.validate!(true)
+
+        assert_equal in_relation_to_request_id, logoutresponse.in_response_to
+
+        assert logoutresponse.success?
+        assert_empty logoutresponse.errors
+      end
+
+      it "validate the response extended" do
+        in_relation_to_request_id = random_id
+        custom_settings = settings
+        custom_settings.idp_entity_id = 'http://app.muda.no'
+
+        logoutresponse = OneLogin::RubySaml::Logoutresponse.new(valid_response({:uuid => in_relation_to_request_id}), custom_settings)
```

Comment:
This would be more readable if the `valid_response` assignment was on its own line.